jaq-core/src/lib.rs

```diff
@@ -55,14 +55,18 @@ pub fn core() -> impl Iterator<Item = (String, usize, Native)> {
 
 fn run<'a>(fs: &'a [(&str, usize, RunPtr)]) -> impl Iterator<Item = (String, usize, Native)> + 'a {
     fs.iter()
-        .map(|(name, arity, f)| (name.to_string(), *arity, Native::new(*f)))
+        .map(|(name, arity, f)| ((*name).to_string(), *arity, Native::new(*f)))
 }
 
 fn upd<'a>(
     fs: &'a [(&str, usize, RunPtr, UpdatePtr)],
 ) -> impl Iterator<Item = (String, usize, Native)> + 'a {
     fs.iter().map(|(name, arity, run, update)| {
-        (name.to_string(), *arity, Native::with_update(*run, *update))
+        (
+            (*name).to_string(),
+            *arity,
+            Native::with_update(*run, *update),
+        )
```

Comment:
I do not like this change; it makes the code more verbose.

Reply:
It is much better in https://github.com/01mf02/jaq/pull/124/commits/8811635174522999fb72c124d53d6eaf9aaa3eb1 now :D